### PR TITLE
Improve wording of contract error messages

### PR DESCRIPTION
**Pull request by graup**

### CODE REVIEW

File: contract/vm_callback.go [hunk 1]
```diff
@@ -267,22 +267,22 @@ func LuaDelegateCallContract(L *LState, service *C.int, contractId *C.char,
...
+		luaPushStr(L, "[Contract.LuaDelegateCallContract] newExecutor error: "+ce.err.Error())
 		return -1
 	}
 
 	var ci types.CallInfo
 	ci.Name = fnameStr
 	err = getCallInfo(&ci.Args, []byte(argsStr), cid)
 	if err != nil {
-		luaPushStr(L, "[Contract.LuaDelegateCallContract] invalid args:"+err.Error())
+		luaPushStr(L, "[Contract.LuaDelegateCallContract] invalid argyments: "+err.Error())
```
Comment (eve2adam) [inline contract/vm_callback.go]: typo

File: contract/vm_callback.go [hunk 2]
```diff
@@ -692,11 +692,11 @@ func LuaECVerify(L *LState, msg *C.char, sig *C.char, addr *C.char) C.int {
 	} else {
 		sign, err := btcec.ParseSignature(bSig, btcec.S256())
 		if err != nil {
-			luaPushStr(L, "[Contract.LuaEcVerify]Error signature parsing:"+err.Error())
+			luaPushStr(L, "[Contract.LuaEcVerify] error parsing signature: "+err.Error())
 			return -1
 		}
 		if pubKey == nil {
-			luaPushStr(L, "[Contract.LuaEcVerify]not supported")
+			luaPushStr(L, "[Contract.LuaEcVerify] pubKey cannot be empty")
```
Comment (eve2adam) [inline contract/vm_callback.go]: pubKey is not input. this error caused by invalid or unsupported address format that can not get pubkey

**Comment by graup** [comment]
Thanks for catching those, I added another commit.

File: contract/vm_callback.go (hunk 1 above)
Comment (graup) [reply contract/vm_callback.go]: fixed

File: contract/vm_callback.go (hunk 2 above)
Comment (graup) [reply contract/vm_callback.go]: is "error recovering pubKey" ok?

Comment (eve2adam) [reply contract/vm_callback.go]: ok

**Pull request merged**